packages/react-components/src/components/CaptionsBanner.tsx

```diff
@@ -50,16 +52,30 @@ export interface _CaptionsBannerProps {
    * @defaultValue 'default'
    */
   formFactor?: 'default' | 'compact';
+  captionsOptions?: {
+    height: 'full' | 'default';
+  };
 }
 
+const scrollOffsetAllowance = 20;
```

Comment:
Change to a constant. 
Can't recall if we have a constant file as well but if not we can keep here but at the top of the file.
```suggestion
const SCROLL_OFF_SET_ALLOWANCE = 20;
```

Reply:
Nice catch! There is constant file, but not in this project